scripts/manual_testing.R

```diff
@@ -10,13 +10,22 @@ library(plumber)
 test_that("custom swagger file update function works", {
   pr <- plumber$new()
   pr$handle("GET", "/:path/here", function(){})
-
+  pr$setApiHandler(function(spec) {
+    spec$info$title <- Sys.time()
+    spec
+  })
+  pr$run(
+    port = 1234,
+    ui = "swagger"
```

Comment:
Let's follow pattern of how `api` is set outside of the run method. 

Trying to keep the run method the bare minimum.

Ex: 
```r
pr$setUI("redoc")
pr$setUI(FALSE)
pr$setUI("redoc", scrollYOffset = 250, disableSearch = TRUE)
```

Something like this to maintain the args and ui value:
```r
setUI = function(ui, ...) {
  private$ui_info = list(
    ui = ui,
    args = list(...)
  )
}
```

Reply:
While I was at it, I think I prefer having different parameter for enabling ui and selecting it. See `setUI` signature.